design/help_overview.md

```diff
@@ -212,9 +212,51 @@ pywbemcli server profiles --help
 b'Usage: pywbemcli server profiles [COMMAND-OPTIONS]\n\n  Display profiles on the current WBEM Server.\n\n  This display may be filtered by the optional organization and profile name\n  options\n\nOptions:\n  -o, --organization <org name>   Filter by the defined organization. (ex. -o\n                                  DMTF\n  -n, --profilename <profile name>\n                                  Filter by the profile name. (ex. -n Array\n  --help                          Show this message and exit.\n'
 ```
 
+pywbemcli connection --help
+===========================
+
+```
+b"Usage: pywbemcli connection [COMMAND-OPTIONS] COMMAND [ARGS]...\n\n  Command group to manage WBEM connections.\n\n  These command allow viewing and setting connection information.\n\n  In addition to the command-specific options shown in this help text, the\n  general options (see 'pywbemcli --help') can also be specified before the\n  command. These are NOT retained after the command is executed.\n\nOptions:\n  --help  Show this message and exit.\n\nCommands:\n  export  Export the current connection information.\n  select  Select a connection from the current defined...\n  set     Show the current connection information, i.e.\n  show    Show the current connection information, i.e.\n  test    Execute a simple wbem request to test that...\n"
+```
+
+pywbemcli connection show --help
+================================
+
+```
+b'Usage: pywbemcli connection show [COMMAND-OPTIONS]\n\n  Show the current connection information, i.e. all the variables that make\n  up the current connection\n\nOptions:\n  --help  Show this message and exit.\n'
+```
+
+pywbemcli connection export --help
+==================================
+
+```
+b'Usage: pywbemcli connection export [COMMAND-OPTIONS]\n\n  Export  the current connection information.\n\n  Creates an export statement for each connection variable and outputs the\n  statement to the conole.\n\nOptions:\n  --help  Show this message and exit.\n'
```

Comment:
This results in a very long line, not only in the source file but also in the rendered output. Split accordingly.